lua/weapons/e.r.c.s.lua

```diff
@@ -0,0 +1,97 @@
+SWEP.Author = "el_patito_loco"
+SWEP.PrintName = "E.R.C.S" --"Experimental Rotating Chamber Sniper"
+SWEP.Category = TASWeapons.Category
+SWEP.Author = TASWeapons.Authors.El_Patito
+SWEP.Instructions = [[Left Click shoots a powerfull 12.7 mm sniper round that will destroy your enemies!!
+At the moments our engineers are working on new features wich will provide more mass destruction!!
+]]
+
+SWEP.Spawnable = true
+SWEP.AdminOnly = false
+
+SWEP.Primary.ClipSize = 10
+SWEP.Primary.DefaultClip = 10
+SWEP.Primary.Automatic = true
+SWEP.Primary.Ammo = "357"
+
+SWEP.Secondary.ClipSize = -1
+SWEP.Secondary.DefaultClip = -1
+SWEP.Secondary.Automatic = true
+SWEP.Secondary.Ammo = "none"
+
+SWEP.Weight = 5
+SWEP.AutoSwitchTo = false
+SWEP.AutoSwitchFrom = false
+
+SWEP.Slot = 4
+SWEP.SlotPos = 3
+SWEP.DrawAmmo = false
+SWEP.DrawCrosshair = true
+
+SWEP.ViewModel = "models/weapons/v_357.mdl"
+SWEP.WorldModel = "	models/weapons/w_357.mdl"
+
+local ShootSound = Sound("weapons/mortar/mortar_fire1.wav")
+
+SWEP.LoadSound = "weapons/ammopickup.wav"
+
+SWEP.Primary.Recoil = 1
+SWEP.Primary.Damage = 75
+SWEP.Primary.NumShots = 1
+SWEP.Primary.Spread = 0
+SWEP.Primary.Cone = 0.25
```

Comment:
`Cone` is not a property of `SWEP.Primary` and you do not use it anywhere else, could this be remnants of old code?

Reply:
In a future when I add the scope and the secondary fire I will activate it but for now its just unused